src/SplashScreen/SplashNav/SplashNav.tsx

```diff
@@ -1,39 +1,40 @@
 import React from 'react';
```

Comment:
These `styled` files do not have to end in `.tsx` they can and probably should end in `.ts` I don't think you need to change them but just a heads up. Any file that contains `jsx` MUST have a file extension that ends in `.jsx` or `.tsx` if the file does not contain any JSX then it can just have the extension`.ts` or `.js`